unity-renderer/Assets/DCLPlugins/SentryPlugin/Tests/SentryControllerTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using DCL;
+using NSubstitute;
+using NUnit.Framework;
+using Sentry;
+using Sentry.Unity;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCLPlugins.SentryPlugin.Tests
+{
+    public class SentryControllerTests
+    {
+        [Test]
+        public void Player_Update_Updates_Sentry_Tags()
+        {
+            // Arrange
+            var sentryHubMock = Substitute.For<IHub>();
```

Comment:
You could move all configuring logic to the `[Setup]` method. It would sanitize the logic of the tests themselves.